packages/fxa-settings/src/models/Account.ts

```diff
@@ -623,10 +623,41 @@ export class Account implements AccountData {
     return result;
   }
 
+  /**
+   * TODO: Remove this once we update the GQL endpoint to accept the
+   *  `accountResetWithRecoveryKey` option and fix graphql-api not reporting
+   *  the correct IP address.
+   *
+   * @param token passwordForgotToken
+   * @param code code
+   * @param accountResetWithRecoveryKey is account being reset with recovery key?
+   * */
+  async passwordForgotVerifyCode(
```

Comment:
Both places that we call this are wrapped in a `try` that fallback to "Unknown error" so we should be safe from an unhandled error here.